docs/operations/hotplug_interfaces.md

```diff
@@ -120,6 +127,31 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+>**Note**: This step can be done when both VM and VMI objects has the new interface in their spec.
+
+### Remove interface
+```bash
+virtctl removeinterface vm-fedora --name dyniface1
+```
+At this point the new interface should be detached from the guest but exist in the pod.
```

Comment:
I think `new` does not fit here.

... but **still** exist in the pod.

Reply:
Fixed